tests/features/repo.feature

```diff
@@ -50,6 +50,19 @@ Feature: Subcommand: dab repo
 		And the output should match /^dotfiles3\s*|\s*not downloaded/
 		And the output should match /^dotfiles4\s*|\s*downloaded/
 
+	Scenario: Can filter listed repositories
+		Given a file named "~/.config/dab/repo/dotfiles3/url" with:
+		"""
+		https://github.com/Nekroze/dotfiles.git
+		"""
+		And the directory "~/dab/dotfiles3/.git/" should not exist
+		And I successfully run `dab repo add dotfiles4 https://github.com/Nekroze/dotfiles.git`
+
+		When I successfully run `dab repo list downloaded`
+
+		Then the output should match /^REPO\s*|\s*STATUS/
+		And the output should match /^dotfiles4\s*|\s*downloaded/
```

Comment:
from what I can see this would also pass without the code change to the list subcommand, you could add another line to assert that the output should not contain the not downloaded status string to do the negative assertion that will ensure this new functionality works.